pkg/cover/heatmap.go

```diff
@@ -169,12 +168,28 @@ where namespace=$1 and dateto>=$2 and dateto<=$3
 	return res, nil
 }
 
-func DoHeatMap(w io.Writer, projectID, ns string, dateFrom, dateTo civil.Date) error {
-	style, body, js, err := DoHeatMapStyleBodyJS(projectID, ns, dateFrom, dateTo)
+func DoDirHeatMap(w io.Writer, projectID, ns string, dateFrom, dateTo civil.Date) error {
+	return DoHeatMap(w, projectID, ns, dateFrom, dateTo, "dir")
+}
+
+func DoSubsystemsHeatMap(w io.Writer, projectID, ns string, dateFrom, dateTo civil.Date) error {
+	return DoHeatMap(w, projectID, ns, dateFrom, dateTo, "subsystems")
+}
+
+func DoHeatMap(w io.Writer, projectID, ns string, dateFrom, dateTo civil.Date, hmType string) error {
+	var style template.CSS
```

Comment:
I wonder if it would be cleaner to duplicate a few lines of code and provide two implementations of `DoHeatMap`.

Reply:
Done.